### PR TITLE
Add Strela-2 and 2M to HDS

### PR DESCRIPTION
**Pull request by Des-mundo**
Strela-2 and 2M were created by cloning the Strela-3 entry and tweaking speeds, ranges, seeker parameters, masses, ccm, physical dimensions, warhead yield, etc. to match researched properties of the 9M32/M systems.

Limited testing shows capability close to that expected by reading after-action testimony and reports from early 70s: 

- strela-2 loves flares
- able to hit gently maneuvering, low flying jets that aren't dropping flares
- contact fuze works as expected
- low damage output, based on the 0.37 kg of explosive in the warhead makes hits mostly survivable with severed hyd/fuel lines or engine failures

ToDos:

1. lock-on aspect limitation for the 9M32: rear only
2. lock-on aspect limitation for the 9M32M: rear and rudimentary side
3. motors: booster, march, march - verify impulse, propellant mass, use of t_acc / t_marsh against this section
4. check if sensor / missile capability expressed in angular tracking speed is implemented (other than by Nr_max) and how; apply researched values
5. explore warhead yield calculations: why calculate piercing_mass if for all ED weapons piercing_mass = mass * 0.2
6. warhead yield through other_factors: does this parameter affect explosion size in the air? ED documentation points to ground effect only

### CODE REVIEW

File: Mods/tech/HighDigitSAMs/Database/Weapon/9M32.lua [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+		target_H_min	= 0.0,
+		flag_dist		= 150.0,
+		abs_err_val		= 3,
+		ground_err_k	= 3,
+		ccm_k0 			= 1.5, --1.0
+	},
+	
+	simple_gyrostab_seeker = {
+		gimbal_lim = math.rad(40)
+		omega_max	= math.rad(8)
```
Comment (Nosajthedevil) [inline Mods/tech/HighDigitSAMs/Database/Weapon/9M32.lua:190]: I couldn't get DCS to load this until I added a comma between the gimbal_lim and omega_max entries. Same for the -2M. Then it seemed to work. Thank you for adding this BTW!

File: Mods/tech/HighDigitSAMs/Database/Weapon/9M32M.lua [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+		target_H_min	= 0.0,
+		flag_dist		= 150.0,
+		abs_err_val		= 3,
+		ground_err_k	= 3,
+		ccm_k0 			= 1.4, --1.0
+	},
+	
+	simple_gyrostab_seeker = {
+		gimbal_lim = math.rad(40)
+		omega_max	= math.rad(8)
```
Comment (Nosajthedevil) [inline Mods/tech/HighDigitSAMs/Database/Weapon/9M32M.lua:190]: I couldn't get DCS to load this until I added a comma between the gimbal_lim and omega_max entries. Same for the -2. Then it seemed to work. Thank you for adding this BTW!

File: Mods/tech/HighDigitSAMs/Database/Weapon/9M32.lua (hunk 1 above)
Comment (Auranis) [reply Mods/tech/HighDigitSAMs/Database/Weapon/9M32.lua:190]: I'll make this change after the merge. Thanks for calling it out @Nosajthedevil

File: Mods/tech/HighDigitSAMs/Database/Weapon/9M32M.lua (hunk 2 above)
Comment (Auranis) [reply Mods/tech/HighDigitSAMs/Database/Weapon/9M32M.lua:190]: Same as above, thanks for the note.

**Pull request merged**